app/src/main/java/org/schabi/newpipe/NewVersionWorker.kt

```diff
@@ -42,26 +44,30 @@ class NewVersionWorker(
         versionCode: Int
     ) {
         if (BuildConfig.VERSION_CODE >= versionCode) {
+            if (inputData.getBoolean(IS_MANUAL, false)) {
+                // Show toast stating that the app is up-to-date if the update check was manual.
+                Toast.makeText(applicationContext, R.string.app_update_unavailable_toast, Toast.LENGTH_SHORT).show()
```

Comment:
No lines longer than 100 characters... ktlint seems to be misconfigured, it should detect these problems
```suggestion
                Toast.makeText(applicationContext, R.string.app_update_unavailable_toast,
                        Toast.LENGTH_SHORT).show()
```